### DISCUSSION TITLE
PyPy3.exe & PyPy.exe

### DISCUSSION THREAD

**Discussion by jmac150** [python-pillow/Pillow#6279] [Q&A] [answered]
Hi
Python newbie here. Downloaded pypy3.9-v7.3.9-win64.zip and extracted to folder, which I added to PATH.
Noticed that there are pypy.exe and pypy3.exe files. The pypy.exe runs fine with python program (Recuperabit in this case) but the pypy3.exe will not launch. What am i missing?
Thanks

**Comment by radarhere** [comment, marked as answer]
When you say it "will not launch", are there any more details that you can offer? A specific error? You're trying to run it from the command line, yes?

As an aside, this is the location for questions regarding Pillow, a specific Python library. Some of us here are Windows users, but not all - for example, I am not. If you have general Python questions, StackOverflow might be a better place for them - https://stackoverflow.com/questions/tagged/python

**Reply by jmac150** [reply]
Hi
Thanks for quick response. By “will not launch”, I mean that after entering pypy3 command, cursor just returns to command prompt. If I use pypy, program launches. 
I will try StackOverflow. Thanks again.

**Comment by hugovk** [comment]
If pypy.exe is also version 3.9 I'd just use that, but you should rather ask Stack Overflow or PyPy for help about how to use PyPy:

https://doc.pypy.org/en/latest/#contact-1

**Comment by jmac150** [comment]
Hi
Thanks for quick response. By “will not launch”, I mean that after entering pypy3 command, cursor just returns to command prompt. If I use pypy, program launches.
I will try StackOverflow. Thanks again.



Sent from mobile device
________________________________
